src/app/site/profile/components/order-lines/order-lines.component.ts

```diff
@@ -92,7 +91,9 @@ export class OrderLinesComponent implements OnInit {
     let statusText = '';
     arr.forEach(el => {
       tickets = el.tickets;
-      statusText = OrderStatus.filter(os => os.status === tickets[tickets.length - 1].status)[0].title;
+      if (tickets.length)
+        statusText = OrderStatus.filter(os => os.status === tickets[tickets.length - 1].status)[0].title;
```

Comment:
Reading first item from empty list throw program to error unless list cannot be empty